tests/test_playbooks/fixtures/global_parameter-0.yml

```diff
@@ -133,13 +129,15 @@ interactions:
     uri: https://foreman.example.org/api/common_parameters
   response:
     body:
-      string: '{"created_at":"2020-10-16 13:53:30 UTC","updated_at":"2020-10-16 13:53:30
-        UTC","hidden_value?":false,"hidden_value":"*****","id":11,"name":"TheAnswer","parameter_type":"string","value":42}'
+      string: '{"created_at":"2023-08-24 07:13:28 UTC","updated_at":"2023-08-24 07:13:28
+        UTC","hidden_value?":false,"hidden_value":"*****","id":9,"name":"TheAnswer","parameter_type":"string","value":"42"}'
```

Comment:
funnily, it seems that Foreman actually did accept an *integer* here, while the type was explicitly set to string :upside_down_face: